boot/initramfs.go

```diff
@@ -125,3 +127,61 @@ func MockInitramfsReboot(f func() error) (restore func()) {
 func InitramfsReboot() error {
 	return initramfsReboot()
 }
+
+// This function implements logic that is usually part of the
+// bootloader, but that it is not possible to implement in, for
+// instance, piboot. See handling of kernel_status in
+// bootloader/assets/data/grub.cfg.
+func updatePibootKernelStatus(bl bootloader.NotScriptableBootloader) error {
```

Comment:
should this function name now refer to nonscripatable instead of piboot?

Reply:
Right, will change in next push